src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -512,7 +529,22 @@ class ChatSwitcherView extends React.Component {
                     onRemoveFromGroup={this.removeUserFromGroup}
                     onConfirmUsers={this.startGroupChat}
                 />
-
+                <View style={[
+                    styles.flexRow,
+                    styles.sidebarHeaderTop,
+                    styles.justifyContentBetween,
+                    styles.alignItemsCenter,
+                ]}
+                >
+                    <Text numberOfLines={2} style={[styles.headerText, styles.textLarge]}>
+                        Chats
+                    </Text>
+                    <AvatarWithIndicator
+                        indicatorPosition="bottom-right"
+                        source={this.props.myPersonalDetails.avatarURL}
+                        isActive={this.props.network && !this.props.network.isOffline}
+                    />
+                </View>
```

Comment:
Can this entire section added please be moved to `SidebarLinks.js` above the `ScrollView` here?

https://github.com/Expensify/Expensify.cash/blob/ca2fa88a5789b82463d35eddc3d57f70a7286868/src/pages/home/sidebar/SidebarLinks.js#L98-L104

The reason is that the `ChatSwitcherView` is a separate component and we will be removing it eventually.

Reply:
ok. I was going to put it there but then I see that there will be feedback messages from the search bar which should come immediately after the search bar but if we are going to remove it then it's fine. I will move that part.